daffodil-runtime1/src/main/scala/org/apache/daffodil/infoset/XMLTextEscapeStyle.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.daffodil.infoset
+/**
+ * XMLTextEscapeStyle determines how to wrap values of elements of type xs:string
+ *
+ * Standard - Special characters ('backslash','quotation mark','ampersand','less-than','greater-than')
+ * in the values of elements of type xs:string are escaped, while,
+ * non-special character are written as is.
```

Comment:
Please fix singular/plural agreement and punctuation:

```
 * Standard - Special characters (backslash, quotation mark, ampersand, less-than, greater-than) in
 * the text of xs:string elements are escaped, while non-special characters are written as is.
 *
 * CDATA - The text of xs:string elements are wrapped in CDATA tags.
```

The same fixes should also be applied to 
- daffodil-japi/src/main/java/org/apache/daffodil/japi/infoset/XMLTextEscapeStyle.java
- daffodil-sapi/src/main/scala/org/apache/daffodil/sapi/infoset/XMLTextEscapeStyle.scala